projects/packages/external-media/src/shared/sources/google-photos/style.scss

```diff
@@ -0,0 +1,170 @@
+@import '@automattic/jetpack-base-styles/gutenberg-base-styles';
```

Comment:
Separated the styles from the [editor/editor.scss](https://github.com/Automattic/jetpack/pull/41282/files#diff-be7109b2a4cce65657c706fe151f7b27b53a17cab32d7eff0571987f8959fbd2)